mesheryctl/internal/cli/root/system/error.go

```diff
@@ -23,6 +23,7 @@ const (
 	ErrProcessingMctlConfigCode     = "1025"
 	ErrRestartMesheryCode           = "1026"
 	ErrK8sQueryCode                 = "1041"
+	ErrK8sConfigCode                = "1045" // please check this error code
```

Comment:
@piyushsingariya How are we calculating an error code. Do we have any convention

Reply:
just set the value `"not set"` and it would be automatically taken care of. A Layer5 toolkit takes care of this during the CI workflows.